Gemfile

```diff
@@ -92,6 +92,19 @@ group :cable do
   gem "sprockets-export", require: false
 end
 
+group :storage do
+  gem "httparty"
```

Comment:
Looks like "httparty" is only used in two tests to make PUT requests to S3 and GCS.

How do you feel about dropping this dependency and simply using `Net::HTTP::Put` in those two tests?

I could make that change if you agree. Thanks.